src/sql/platform/clipboard/browser/clipboardService.ts

```diff
@@ -0,0 +1,73 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IClipboardService } from 'sql/platform/clipboard/common/clipboardService';
+import { IClipboardService as vsIClipboardService } from 'vs/platform/clipboard/common/clipboardService';
+import { URI } from 'vs/base/common/uri';
+
+export class ClipboardService implements IClipboardService {
+	_serviceBrand: any;
+
+	constructor(
+		@vsIClipboardService private _vsClipboardService: vsIClipboardService
+	) { }
+
+	/**
+	 * Writes the input image as a dataurl to the clipbaord
+	 */
+	async writeImageDataUrl(data: string): Promise<void> {
+		// not implemented until web standards catch up
+		throw new Error('Not implemented');
```

Comment:
I would not throw an error since its unlikely the caller is actually setup to handle this. I would just throw a notification up that copying an image is not supported.

Reply:
Sounds like a good idea :D